### ISSUE TITLE
Bug on moving initialised metric to cuda 

### ISSUE THREAD

**Issue by allanchan339** [chaofengc/IQA-PyTorch#63]
I have the following bugs as depicted. I wonder if .to(device) do not work for pyiqa metric. 

![image](https://user-images.githubusercontent.com/38370649/222947027-e0064ea8-760d-451e-8ad7-9ac852120357.png)

**Comment by allanchan339** [comment]
This is the code FYI
`import pyiqa
import torch

perceptual = pyiqa.create_metric('dists', as_loss=True, device=torch.device('cpu'))
img1 = torch.rand((1,3,256,256))
img2 = torch.rand((1,3,256,256))

img1 = img1.to('cuda:5')
img2 = img2.to('cuda:5')

perceptual = perceptual.to('cuda:5')

loss = perceptual(img1, img2)
`

**Comment by chaofengc** [comment]
Thanks for your info. The bug is fixed at [08f8850](https://github.com/chaofengc/IQA-PyTorch/commit/08f88508f71eaf02234ebd04aeda091fdc3ab8e7)

**Comment by allanchan339** [comment]
Hi @chaofengc ,

How can I get the latest version ? Is pip install available?

**Comment by chaofengc** [comment]
You may use the following commands:
```
# Install latest github version
pip uninstall pyiqa # if have older version installed already 
pip install git+https://github.com/chaofengc/IQA-PyTorch.git
```
